mava/components/updating/terminators.py

```diff
@@ -101,7 +102,7 @@ def on_parameter_server_run_loop_termination(
             and parameter_server.store.parameters[self.termination_key]
             > self.termination_value
         ):
-            print(
+            logging.exception(
```

Comment:
I tried to use `logging.exception`, it shows the message but also it shows another msg `NoneType: None` for example:
```
[evaluator/0] E1108 11:08:43.800992 140064315311936 environment_loop.py:299] Terminate the system
[evaluator/0] NoneType: None
```
And if I raise an error it shows only the error without the msg
```
[evaluator/0] Traceback (most recent call last):
[evaluator/0]   File "/home/omayma/Desktop/Mava/mava/environment_loop.py", line 281, in run
[evaluator/0]     step_executor()
[evaluator/0]   File "/home/omayma/Desktop/Mava/mava/environment_loop.py", line 265, in step_executor
[evaluator/0]     raise ValueError
[evaluator/0] ValueError
```

Reply:
Only the tf.print but we can keep this one for now and make an issue or more investigation in the future if we can find jax debug print or something similar

Reply:
Sounds good.